Travis complains about unsastified dependency in measure.js

Has something changed ? 

info closure-util Getting Closure dependencies
ERR! closure-util Unsatisfied dependency "ol.FeatureOverlay" in script: /home/travis/build/Geoportail-Luxembourg/geoportailv3/node_modules/ngeo/src/ol-ext/interaction/measure.js
make: *** [.build/build.js] Error 1

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "less": "~1.7.5",
     "ngeo": "camptocamp/ngeo#master",
     "nomnom": "~1.6.2",
-    "openlayers": "openlayers/ol3#master",
+    "openlayers": "openlayers/ol3#v3.6.0",
     "proj4": "2.3.3",
     "phantomjs": "~1.9.7-5",
     "typeahead.js": "~0.10.5",

